projects/js-packages/publicize-components/src/components/post-publish-one-click-sharing/index.js

```diff
@@ -0,0 +1,31 @@
+import { Text, ThemeProvider } from '@automattic/jetpack-components';
+import { JetpackEditorPanelLogo } from '@automattic/jetpack-shared-extension-utils';
+import { PluginPostPublishPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { ShareButtons } from '../share-buttons/share-buttons';
+import styles from './styles.module.scss';
+
+const PostPublishOneClickSharing = () => {
+	return (
+		<>
```

Comment:
We don't need a fragment here, do we?

Reply:
Not now, I was playing with adding the icon, and just removed that to create this draft PR so you can test your change, and was left in there 😄